django/thunderstore/core/urls.py

```diff
@@ -29,7 +30,7 @@
 AUTH_ROOT = "auth/"
 
 urlpatterns = [
-    path("", PackageListView.as_view(), name="index"),
+    path("", CommunitiesView.as_view(), name="index"),
```

Comment:
Hard to say by the context provided here, but are you sure this doesn't affect other pages, i.e. the actual package listings?

Reply:
Please add a comment if you double checked this and it needs no changes.